### DISCUSSION TITLE
BreakupSimulator3DS

### DISCUSSION THREAD

**Discussion by Dangerbites** [lovebrew/lovebrew.github.io#13] [Show and tell]
# Breakup Simulator 3DS
![Art for breakup sim portable, mad eby haynster](https://img.itch.zone/aW1hZ2UvMjUyODQ3My8xNTA0NjM1MS5wbmc=/347x500/BtrjKY.png)

## Short Description
In this quirky dating game, your mission is to break up with Cat Grill as quickly as possible! Make choices to lower the emotion meter before the timer runs out. Can you handle the pressure?

## Author(s)
- **DangerBites** - Re-programming for port, art, and music
- **Haynster** - Original game programming and music
- **Eran** - Creator of the drawing held by Cat Grill

## GitHub Link
[Breakup Simulator 3DS Repository](https://github.com/Dangerbites/BreakupSimulator3DS)

## Supported Platforms
- Nintendo 3DS

## Additional Links
- [Play the Portable Version on Itch.io](https://dangerbites.itch.io/breakup-portable)

**Comment by TurtleP** [comment]
Ah, I didn't see the notification for this. I'll add it tomorrow to the showcase. Thank you :)
